server/authentication/strategies.js

```diff
@@ -0,0 +1,58 @@
+const fiveMinutesBefore = require('../utils/fiveMinutesBefore');
+const {getIn} = require('../utils/functionalHelpers');
+const allowedRoles = require('./roles');
+const logger = require('../../log');
+
+module.exports = function(signInService, userService, audit) {
```

Comment:
I wouldn't change it now, but this looks to be over engineered.  Factoring out the strategies into their own file would be good if the feature switch is designed to be long lived and we be using both for a long period of time.  In our case it makes it harder to take the code out when the feature switch is removed.

Reply:
Discussed and renamed file